docs/docsite/rst/conf.py

```diff
@@ -386,3 +392,22 @@
 ]
 linkcheck_workers = 25
 # linkcheck_anchors = False
+
+# Generate redirects for pages when building on Read The Docs
+def setup(app: Sphinx) -> dict[str, str]:
+
+    if tags.has('redirects'):
```

Comment:
Within `setup()`, you don't actually need to rely on the magically injected var `tags`. It's better to do
```suggestion
    if 'redirects' in app.tags:
```
here.